--- a/mxSetup.py	2015-08-14 17:04:13.000000000 +0200
+++ b/mxSetup.py	2026-08-03 02:00:45.846412176 +0200
@@ -5023,7 +5023,7 @@
 
         if self.install_dir is None:
             installobj = self.distribution.get_command_obj('install')
-            self.install_dir = installobj.install_data
+            self.install_dir = installobj.install_platlib
         if _debug:
             print 'Installing data files to %s' % self.install_dir
         self.set_undefined_options('install',
